tr/messages.json

```diff
@@ -1 +1,430 @@
-{}
+{
+	"extensionDescription": {
+		"message": "Cümleler arasındaki uzun duraksamaları ileri sararak ~ 1,5 kat daha hızlı izleyin."
```

Comment:
Is the comma good here?

Reply:
I mean, isn't it gonna look a little weird? There is no comma in any other translation.

Reply:
In Turkish it must be comma. You can see an example for different cultures from Microsoft's document below.

![image](https://user-images.githubusercontent.com/41087670/173106875-6d969fc6-47d7-49ea-a15e-332c9626fc5e.png)

Reply:
Oh my god I mixed up "comma" with "period". I meant the period at the end of the string.

Reply:
Alright, I'll remove it for now, you can put it back if it's more correct.

Reply:
You can remove it doesn't mater.